src/Uno.UI/Microsoft/UI/Xaml/Controls/PipsPager/PipsPagerTemplateSettings.Properties.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class PipsPagerTemplateSettings : DependencyObject
+	{
+		public IList<object> PipsPagerItems
+		{
+			get => (IList<object>)GetValue(PipsPagerItemsProperty);
+			set => SetValue(PipsPagerItemsProperty, value);
+		}
+
+		public static DependencyProperty PipsPagerItemsProperty { get; } =
```

Comment:
From WinUI sources, this is public. But what I'm seeing in a C# app is it's not public.

![image](https://user-images.githubusercontent.com/31348972/134733799-821a4bac-a127-4a0a-bdd8-caf33a2bf1b1.png)

No idea what's going on.